src/ppspp_options.erl

```diff
@@ -228,8 +239,8 @@ get_chunk_addressing_method(Options) ->
 %% @end
 -spec get_content_integrity_check_method(options()) -> any().
 
-get_content_integrity_check_method(Options) ->
-    get(content_integrity_check_method, Options).
+get_content_integrity_check_method(#{content_integrity_check_method := Method} = _Options) ->
+    Method.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% @doc get_merkle_hash_function/1 returns the hash function for the provided swarm
```

Comment:
Line 246 is too long: %% @doc get_merkle_hash_function/1 returns the hash function for the provided swarm.